lib/populator.js

```diff
@@ -24,6 +24,16 @@ function Populator(options) {
   });
 }
 
+// Only adds the dhis version to DHIS URL if it is specified. This ensures backwards compatibility.
+// Needed as DHIS v2.29 api has breaking changes so we need to use the older apis
+function createDhisUrl(dhisVersion, dhisPath) {
+  let tempPath = dhisPath
+  if (dhisVersion) {
+    tempPath = `${dhisVersion}/${tempPath}`
+  }
+  return `/api/${tempPath}`
```

Comment:
There shouldn't be a leading "/" here so that it matches the previous behaviour.